src/app/search/flows/initial-viewer-search.ts

```diff
@@ -14,7 +15,7 @@ import {viewerSearch} from "./viewer-search"
 
 const initialViewerSearch = (): Thunk<any> => (dispatch, getState) => {
```

Comment:
with the work in your viewer PR I think this will work even better by receiving an already formatted/toString'd query so it won't need to handle this pin logic.

Reply:
correct

Reply:
In fact, all these files have been deleted.